telco-ran/configuration/argocd/example/acmpolicygenerator/README.md

```diff
@@ -145,3 +145,13 @@ includes the statically generated Policy) or PolicyGenerator.
 ### Other
 
 The pgt2acmpg supports converting Policy Gen Templates to ACM Policy Generator templates. More details can be found at [link](https://github.com/openshift-kni/cnf-features-deploy/ztp/tools/pgt2acmpg/blob/main/README.md)
+
+## Controlled reboot a fleet of clusters
+
+Users are able to make tuning changes by applying tuned configurations on a running system. When the tuning change is done, the tuned process rollbacks all the configurations and reapplies them. In some cases, latency sensitive applications can't tolerate the removal/re-apply of the tuned profile. By adding `tuned.openshift.io/deferred` annotation to the `Tuned` CR, applying of the configuration can be deferred to the maintenance window. After rebooting the node the deferred tuning configuration will be applied to the node.
+
+PolicyGenerator `acm-example-reboot.yaml` can be used to reboot clusters. The policy generated contain a `MachineConfig` which will trigger a reboot when reconciled and a `MachineConfigPool` (MCP) validator which verifies that the MCP is successfully updated. These policies can be rolled out using `CGU` to desired clusters. In case of multi-node clusters the `MCP` should match the `Tuned.spec.recommended`. Note that all the nodes belong to the `MCP` at the time of rolling out the policy will be rebooted.
```

Comment:
Some suggestions:
`The policy generated contains`
`Note that all the nodes belonging to the `MCP` at the time when the policy is rolled out will be rebooted.1